packages/mobile/.editorconfig

```
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = false
+insert_final_newline = false
```

Looks like we should [trim trailing whitespace](https://softwareengineering.stackexchange.com/a/298050) and [insert final newline](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file). What do you think?

```suggestion
trim_trailing_whitespace = true
insert_final_newline = true
```